### ISSUE TITLE
Exception throws when running whole suite

### ISSUE THREAD

**Issue by igerasimov23** [BorisOsipov/wdio-reportportal-reporter#9]
## The problem

Error throws `TypeError: Cannot read property 'type' of null` when integrated with whole testing suite. Test were on Jenkins + Saucelab

## Environment

* WebdriverIO version: 4.11
* wdio-reportportal-version version: 0.9
* Node.js version: v8.10.0
* WDIO Testrunner and version(mocha\cucumber\jasmine):
* Additional wdio packages used (if applicable): mocha

## Details

Following error throws during whole suite execution: 

mac_stg_webriderIO_sauce_debugv2/api/test/e2e/node_modules/webdriverio/build/lib/utils/BaseReporter.js:336
                        throw _iteratorError;
                        ^

TypeError: Cannot read property 'type' of null
    at ReportPortalReporter.testStart (/var/nyt/jenkins-slave/workspace/qa/qa-[*******]/qa-[*******]-mac_stg_webriderIO_sauce_debugv2/api/test/e2e/node_modules/wdio-reportportal-reporter/build/reporter.js:121:16)
    at emitOne (events.js:116:13)
    at ReportPortalReporter.emit (events.js:211:7)
    at BaseReporter.handleEvent (/var/nyt/jenkins-slave/workspace/qa/qa-[*******]/qa-[*******]-mac_stg_webriderIO_sauce_debugv2/api/test/e2e/node_modules/webdriverio/build/lib/utils/BaseReporter.js:324:35)
    at Launcher.messageHandler (/var/nyt/jenkins-slave/workspace/qa/qa-[*******]/qa-[*******]-mac_stg_webriderIO_sauce_debugv2/api/test/e2e/node_modules/webdriverio/build/lib/launcher.js:688:28)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:772:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)

## Code To Reproduce Issue [ Good To Have ]

wdioconfig.js
```
reportportal: {
      rpConfig: {
        token: 'xxx-xxx',
        endpoint: 'xxxxx',
        launch: 'launch_name',
        project: 'cooking',
        mode: 'DEFAULT'
      },
      enableSeleniumCommandReporting: true,
      enableScreenshotsReporting: false,
      seleniumCommandsLogLevel: 'debug',
      screenshotsLogLevel: 'info',
      enableRetriesWorkaround: true
```

**Comment by igerasimov23** [comment]
Hello,
Took me a while I narrowed down to  the test cases where it was failing. Looks like you are not handling properly case where user would have `test.skip`:

` it.skip('my test', function () {
      //will fail here
    });`

Let me know if you need more info from me

**Comment by BorisOsipov** [comment]
@igerasimov23 Fixed in 0.0.10, can you check?

**Comment by igerasimov23** [comment]
Retested. Works great!
